action.yml

```diff
@@ -9,8 +9,14 @@ inputs:
         description: 'TBD: Name of the Container Group Instance - this should be aquired with cred automatically'
         required: true
     image:
-        description: Name of Container Image
+        description: Name of Container Image. If the `mode` input is `deactivate-revision`, this input is ignored.
         required: true
+    revision-name-suffix:
+        description: Suffixes for New Revision or Target Revision of deactivation. If the `mode` input is `deactivate-revision`, this input needs to be set.
+        required: false
+    mode:
+        description: Mode of this action, `deactivate-revision` or `add-revision`. Default value is `add-revision`
```

Comment:
For example, I'll give you the words as follows
`revisionMode: "deactivate"`
Or how about a boolean like this
`deactivateRevisionMode: true`

Since we have two choices this time, we might not want to pass strings as arguments.

Reply:
Thank you for good suggestion!
I will change soon.

Reply:
Updated.